src/LayoutComponents/nav/collapsible.scss

```diff
@@ -0,0 +1,52 @@
+@mixin menu-transition {
+  transition: transform 250ms ease-in-out, height 250ms ease-in-out, opacity 250ms ease-in-out ;
+}
+
+@mixin open {
+  height: 100%;
+  opacity: 1;
+  transform: translate(0px, 0%) scaleY(1);
+}
+
+@mixin closed {
+  height: 0px;
+  opacity: 0;
+  transform: translate(0px, -45%) scaleY(0.1);
+}
+
+.collapse-button {
+  background-color: transparent;
+  border: none;
+  color: #006400;
+  font-size: 45px;
+}
+
+.collapse-menu {
+  @include menu-transition;
+
+  &.entering {
+    @include closed;
+  }
+  &.entered {
+    @include open;
+  }
+  &.exiting {
+    @include open;
+  }
+  &.exited {
+    @include closed;
+  }
+}
+
+.hidden-menu-bar {
+  opacity: 0.25;
```

Comment:
Maybe this opacity should be a bit higher, as it will mostly be displayed on touch devices where you can't hover.